### DISCUSSION TITLE
Undefined Error after login with 4.10 upgrade

### DISCUSSION THREAD

**Discussion by Nithin-ns24** [Sunbird-Ed/Community#558] [Bugs]
Hello community
we have upgraded our DEV env from 4.4 to 4.10 release by following http://docs.sunbird.org/4.10.0/developer-docs/upgrade/. 
After login to the Application we are facing issue as below.
![image](https://github.com/Sunbird-Ed/Community/assets/99644046/77cf3126-a6df-46e1-9c15-c5615a006d6b)

**undefined : Request URL: https://host_url/api/channel/v1/read/undefined
Request Method: GET
Status Code: 404 Not Found**

![image](https://github.com/Sunbird-Ed/Community/assets/99644046/71ca75d7-fd0a-4713-af6b-10d28d402257)

@santhosh-tg @rajeevsathish could you please look into this and help us to resolve Thank you

cc: @gandham-santhosh

**Comment by chitranshu-keshav** [comment]
@rajeevsathish Can you please get back on this

**Comment by gandham-santhosh** [comment]
@Nithin-ns24 looks like it is not able to fetch the channel from the DB. Can you please check if the default channel id details are part of cassandra and Neo4j?

**Reply by Nithin-ns24** [reply]
Issue is Resolved sir as discussed over a call we copied the organisation table from Prod to Dev env which is missing

**Comment by Nithin-ns24** [comment]
> @Nithin-ns24 looks like it is not able to fetch the channel from the DB. Can you please check if the default channel id details are part of cassandra and Neo4j?
